src/rgw/rgw_op.cc

```diff
@@ -3130,7 +3130,7 @@ void RGWPutMetadataBucket::execute()
   if (has_cors) {
     buffer::list bl;
     cors_config.encode(bl);
-    emplace_attr(RGW_ATTR_CORS, std::move(bl));
+    attrs[RGW_ATTR_CORS] = std::move(bl);
```

Comment:
This is a bit surprising. From the code I can deduce the assumption that `RGW_ATTR_CORS` (and its current value from the RADOS store) shouldn't be present in `RGWPutMetadataBucket::attrs` at this phase. I need to debug this further.